src/BitmapFont.ts

```diff
@@ -55,16 +68,16 @@ export class BitmapFont {
         const result: { [x: string]: number} = {};
         const colors = Object.keys(colorMap);
         const count = colors.length;
-        const w = this.canvas.width = this.sourceImage.width;
-        const h = this.sourceImage.height;
-        this.canvas.height = h * count;
-        this.charHeight = h;
-        const ctx = this.canvas.getContext("2d")!;
+        const size = new Size(this.sourceImage.width, this.sourceImage.height);
+        this.canvas.width = size.width;
+        this.canvas.height = size.height * count;
+        this.charHeight = size.height;
+        const ctx = this.canvas.getContext('2d')!;
```

Comment:
Wow, the battle begins. Single quotes vs double quotes... Who wanna fight? :-) I fight for double quotes and line up some arguments:

* JSON is a part of JavaScript and JSON only supports double quotes. Means you can't simply copy objects from JavaScript into a JSON file when using single quotes.
* Pretty much any other C-like language uses double quotes for strings and I hate it to get used to different string syntax when switching context from Java or C to JavaScript.
* Single quotes are for Delphi pussies! ;-)
* The old argument of having double quotes in HTML so HTML strings in JavaScript must use single quotes is outdated thanks to template strings.